apps/examples/power/power_main.c

```diff
@@ -33,54 +33,51 @@
 /****************************************************************************
  * Pre-processor Definitions
  ****************************************************************************/
-#define PM_LOCK_PATH						"/proc/power/domains/0/pm_lock"
-#define PM_UNLOCK_PATH					"/proc/power/domains/0/pm_unlock"
-#define PM_SLEEP_PATH						"/proc/power/domains/0/enter_sleep"
+#define POWER_THREAD_SIZE				2048
+#define POWER_THREAD_PRIORITY				255
+#define PM_DAEMON_INTERVAL				5
 
-#define POWER_THEAD_SIZE				2048
-#define POWER_THEAD_PRIORITY		255
-#define PM_DAEMON_INTERVAL			15
-#define PM_LOCK(x)								power_set_int(PM_LOCK_PATH, x)
-#define PM_UNLOCK(x)							power_set_int(PM_UNLOCK_PATH, x)
+#define PM_WAKEUP_TIMER_DURATION			5
 
 /****************************************************************************
  * Private Functions
  ****************************************************************************/
-static int power_write_buf(char *file, char *buf)
+#ifdef CONFIG_ARCH_CORTEXA32
+static int power_write_buf(char *file, bool use_timer)
+#else
+static int power_write_buf(char *file)
+#endif
 {
 	int fd;
-	int r;
 	int ret = 0;
 
 	fd = open(file, O_WRONLY);
 	if (fd == -1)
 		return -ENOENT;
```

Comment:
Suggest to add pmdbg(...) to notice user that we have failed to open some file descriptors:
Possible reasons:
- did not enable procfs automount config

Similar as to read/write operation below~

Reply:
Using printf as a debug statement here
Added